addons/website_slides/controllers/main.py

```diff
@@ -240,7 +240,7 @@ def slides_channel_home(self, **post):
         domain = request.website.website_domain()
         channels_all = request.env['slide.channel'].search(domain)
         if not request.env.user._is_public():
-            channels_my = channels_all.filtered(lambda channel: channel.is_member).sorted('completion', reverse=True)[:3]
+            channels_my = channels_all.filtered(lambda channel: channel.is_member).sorted('completion', reverse=True).sorted('completed')[:3]
```

Comment:
Not sure what you want here :
Among all the channels, you want to have only the 3 first that are completed but with the maximum completion ?
I think if completed is True, completion will be 100% anyway.
So you can just add in you filter the completed flag and take the 3 first.

Note that completed is now a boolean (and not the number of completed slides anymore) --> rebase if necessary
Also, sorting two times in a row is useless, the last sorting only will be taken into account.